frontend/src/setupTests.ts

```diff
@@ -0,0 +1,44 @@
+import '@testing-library/jest-dom' // Idiomatic DOM matchers for Jest
+import 'jest-canvas-mock' // Mock Canvas API
+import { Crypto } from '@peculiar/webcrypto'
+
+import 'locales' // Locales necessary for I18nProvider
+import { server } from './test-utils'
+
+// Mock WebCrypto APIs
+// Redeclare the type of `global` as it does not include the `crypto` prop
+interface Global extends NodeJS.Global {
+  crypto: Crypto
+}
+declare const global: Global
+global.crypto = new Crypto()
+
+beforeAll(() => {
+  // Enable API mocking before tests
+  server.listen({ onUnhandledRequest: 'error' })
+
+  // Workaround for issue with 'muted' property not being treated as a default attribute
+  // see: https://github.com/testing-library/react-testing-library/issues/470
+  Object.defineProperty(HTMLMediaElement.prototype, 'muted', {
+    set: jest.fn(),
+  })
+})
+
+beforeEach(() => {
+  // Fake timers using Jest
+  jest.useFakeTimers()
```

Comment:
Not sure about calling this at the global level. It would probably be better to make explicit calls to `jest.useFakeTimers` in the tests itself. Also there might be instances where you want to call `jest.runOnlyPendingTimers` and make assertions after the pending timers have executed.

Reply:
I've removed this line and the corresponding `jest.useRealTimers()` call. Thanks!